CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## 1.8.6
+## [UNRELEASED] neptune 1.9.0
+
+### Features
+- Added fetching with iterators ([#1585](https://github.com/neptune-ai/neptune-client/pull/1585))
```

Comment:
Can we mention `.fetch_*_table()` methods?

Reply:
I don't think it's for efficiency 😉. It was fine except lack of `fetch_*_table` name.

Reply:
The task description mentioned efficiency :wink: 
But let me change it anyway